### ISSUE TITLE
Regarding the different transformations on train and validation set

### ISSUE THREAD

**Issue by Linardos** [pytorch/examples#305]
I am getting a small fraction of better test performance than train. Despite it being small, it was also very consistent and got me thinking

I noticed that the DataLoader performs different transformations on the train images vs validation images
On the train images it does a Random Sized Crop and a Random Horizontal Flip
On the validation images it instead does a Center Crop

Wouldn't that cause the distributions of the 2 datasets to differ? I'd thought it mandatory that when estimating generalization performance the train and test set should come from the same distribution

**Comment by soumith** [comment]
natural images dont have significantly different distributions on random crop vs center crop. they are different (because of how we take pictures, center usually has more information of interest), but not significantly.

in computer vision, it's become standard practice across the industry to do a center crop as a first-check and for more refined results, do a 5-crop (center + {left/right/top/bottom} corner)/ 10-crop (5-crop + hflip) / 144-crop (more dense cropping).

the examples reflect this common practice.
